### DISCUSSION TITLE
Can we add string method like split or replace or regex in data link URL

### DISCUSSION THREAD

**Discussion by lalitdhami007** [grafana/grafana#54615] [Observability]
Can we add string method like split or replace or regex in data link URL 
![image](https://user-images.githubusercontent.com/81741142/188067011-93a38a10-5a14-4302-bfa9-f16a668f43a7.png)
Right now if we want to get substring of a field there is no option to do that we have to do it field.name.0 like that 
if we have a string  Tester | Devloper and i want to pass only Developer as a text there is no option to do that there should be a function like split or other to get substring from a string

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
